src/ctree-app/ctree-landing.html

```diff
@@ -416,28 +517,75 @@
         <div id="overview" class="section-styling">
         </div>
         <div id="feedback" class="section-styling">
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
-          <p>W</p>
+          <div class="feedback-container">
+            <h1>Feedback</h1>
+            <div class="feedback-cards">
+              <div class=feedback-card>
```

Comment:
I recommend creating a new testimonials (feedback is going to be used somewhere else too, and these are really just testimonials) card Polymer element instead of something inlined with a class like this.  One thing I forgot to update on the description for #59 is that work could be based on the [#59-landing-page-feedback](https://github.com/F4IF/ctree-demo/tree/%2359-landing-page-feedback), which was worked on by a developer who got busy with another project and was unable to complete it.  You're free to continue working on your own branch, but may want to look at it for ideas of how to implement the rest of the functionality.

Reply:
will do, this was a work in progress, as i didn't know how to implement the rest of the functionality in terms of polymer. I'll checkout that other branch and see how it works.